config.json

```
@@ -182,7 +185,10 @@
         "slug": "pangram",
         "name": "Pangram",
         "uuid": "4bc9a4f8-fc4d-4e6b-a93c-07069bbc5bc9",
-        "prerequisites": [],
+        "prerequisites": [
+          "boolean-logic", "loops",
```

```suggestion
          "boolean-logic", "loops", "sets"
```

This gives away the solution but I guess it may still make sense to have it as pre-req?

---

That's the question, I guess. You don't need to use sets to solve this (and many students solve it without sets). Sets are just the most elegant solution.

But this one is perfectly fine, so I don't think I'd want to discourage solutions like this:

```jl
ispangram2(input) = all(in(lowercase(input)), 'a':'z')
```

This is why I think a data-structures or iterables concept might be a good idea?